internal/audiorecord/audiorecord.go

```diff
@@ -13,24 +11,16 @@ const (
 )
 
 type Recorder struct {
-	deviceVolume float64
-	lastValue    float64
 }
 
 func Open(gco *coroutine.Coroutines) *Recorder {
-	p := &Recorder{
-		deviceVolume: 0,
-	}
-
-	return p
+	return &Recorder{}
```

Comment:
I suggest to panic here to make it more obvious that we "do not support related APIs such as `Loudness`", just like other unsupported APIs, e.g., `Timer`、`Ask`、`Answer`.